internal/operands/common-templates/resource.go

```diff
@@ -0,0 +1,129 @@
+package common_templates
+
+import (
+	"bytes"
+	templatev1 "github.com/openshift/api/template/v1"
+	"io"
+	"io/ioutil"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"sync"
+)
+
+const (
+	goldenImagesNSname = "kubevirt-os-images"
+	version            = "v0.12.2"
+	bundleDir          = "../../../data/common-templates-bundle/"
+)
+
+var (
+	loadTemplatesOnce sync.Once
+	templatesBundle   []templatev1.Template
+)
+
+func clusterRoleName(role string) string {
+	return "os-images.kubevirt.io" + ":" + role
```

Comment:
I would prefer to have resource names as constant values at the top of the file

Reply:
Changed now